client/state/data-getters/index.js

```diff
@@ -87,6 +87,7 @@ export const requestSiteAlerts = siteId => {
 								type: theme.type,
 								version: theme.version,
 							} ) ),
+							core: updates.core,
```

Comment:
is there more knowledge we have about `updates.core` here that we could make explicit as with `themes` and `threats`?